pkg/continuoustest/config.go

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+
+package continuoustest
+
+import (
+	"flag"
+
+	"github.com/grafana/dskit/log"
+)
+
+type Config struct {
+	ServerMetricsPort   int                       `yaml:"-"`
```

Comment:
This and `LogLevel` doesn't seem needed anymore? If this is still required for the stand-alone binary maybe we should split the configuration.